applications/nftaa.md

```diff
@@ -0,0 +1,411 @@
+|-------:|-----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **0a.** | License                                      | MIT |
+| **0b.** | Documentation                                | Inline documentation of code, as well as startup configuration with all necessary commands, included in repository |
+| **0c.** | Testing and Testing Guide                    | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker                                       | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+|    0e. | Article         | We will publish an **article** that explains NFTAA (what was done/achieved as part of the grant). The article will be published through an online media service, e.g., medium.com, and advertised through social networks. Our ambition, as we are from academic environment, is to have also an academic paper about the idea of NFTAA in Polkadot published in a conference venue                                                                                    |
+|    3a. | Web app         | Develop a web application to interact with the nftaa_pallet, allowing users to create, manage, and transfer NFTAAs, provide a web application to interact with the nftaa_pallet: allowing users to create collections and single item; manage of metadata and attributes; operate with NFTAA ownership (transfers); play with staking, increase stake and unstake handled by NFTAA; read operations as get for an account, get assets of NFTAA, get the owner of NFTAA |
+|    3b. | Marketplace app | We will also create a lightweight marketplace application for easily buying and selling the NFTAAs with an overall list in a form of a bulleting board.                                                                                                                                                                                                                                                                                                                |
```

Comment:
Why is this necessary and wouldn't be supported by existing NFT marketplaces?

Reply:
We can certainly make sales through the Koda platform, for example, and so it's true that we can prepare an integration to them instead of our own web app. We also have a good relationship with them and we believe that this kind of cooperation will not be a problem.

The lightweight application is more for testing purposes.

Reply:
Why do you need this for testing purposes? For your internal testing you can just use bare extrinsics. Once you made sure your NFTs are working as expected it should work like a charm with Koda. Why wouldn't your NFTs be tradeable just like any other NFTs?

Reply:
You're right. A change has already been made and the lightweight application is no longer part of the proposal.
You can see the changes [here](https://github.com/w3f/Grants-Program/pull/2396/commits/1a8e20e1d464cf4d7e29e3da464d0673f3ca191d)